cluster/build.gradle

```diff
@@ -181,41 +181,5 @@ scalaTest {
   }
 }
 check.dependsOn test, scalaTest, dunitTest
-
 archivesBaseName = 'snappydata-cluster_' + scalaBinaryVersion
 
-task generateSnappydataZeppelinInterpreterJar(type: Jar, dependsOn: compileScala) {
-
-  //from sourceSets.main.output.classesDir
-  //from sourceSets.main.output.resourcesDir
-  //from sourceSets.main.resources.srcDirs
- // from 'src/main/resources/snappydata'
-  zip64 = true
-  duplicatesStrategy = DuplicatesStrategy.EXCLUDE
-  from sourceSets.main.output.classesDir
-  jar.includeEmptyDirs = false
-  duplicatesStrategy = DuplicatesStrategy.EXCLUDE
-  from {
-    configurations.compile.collect { it.isDirectory() ? it : zipTree(it) }
-  }
-  from('src/main/resources/snappydata'){
-    include 'interpreter-setting.json'
-    include 'ansi.sql.keywords'
-  }
-  jar.includeEmptyDirs = false
-
-  include 'org/apache/zeppelin/spark/**'
-  include 'org/apache/zeppelin/jdbc/**'
-  include 'org/apache/zeppelin/scheduler/**'
-  include 'org/apache/zeppelin/interpreter/**'
-  include 'com/google/**'
-  include 'org/slf4j/**'
-  include 'jline/console/**'
-  include 'jline/internal/**'
-  include 'jline/**'
-  include 'org/apache/commons/lang/**'
-
-  include 'org/apache/zeppelin/interpreter/**'
- // exclude 'org/apache/zeppelin/interpreter/SnappydataSqlZeppelinInterpreter**'
-  archiveName 'snappydata-zeppelin-interpreter-'+version+'.jar'
-}
```

Comment:
There still are references to zeppelin-interpreter/spark/jdbc in this build file. Are they still required after these changes?